eng/templates/stages/deploy.yaml

```
@@ -141,6 +143,33 @@ stages:
     - powershell: .\eng\common\build.ps1 -restore
       displayName: Install .NET
 
+    - powershell: |
+        mkdir darc
+        
+        $dotnetDir = cmd /c "where dotnet"
+        Invoke-Expression "& '$dotnetDir' tool install Microsoft.DotNet.Darc --prerelease --tool-path .\darc --add-source $(Pipeline.Workspace)\PackageArtifacts" 
+      displayName: Install Darc
+
+    - task: AzureCLI@2
+      name: GetMaestroToken
+      displayName: Get Maestro Token
+      inputs:
+        azureSubscription: ${{ parameters.ScenarioTestSubscription }}
+        addSpnToEnvironment: true
+        scriptType: ps
+        scriptLocation: inlineScript
+        inlineScript: |
+          $token = (az account get-access-token --resource "$env:servicePrincipalId" | ConvertFrom-Json).accessToken
+          echo "##vso[task.setvariable variable=Token;isOutput=true]$token"
```

```suggestion
          echo "##vso[task.setvariable variable=Token;isOutput=true;isSecret=true]$token"
```
Consider marking as `isSecret` to hint that it should not appear in logs.

---

Thanks for the pointer, didn't know about that option!